x/outline-connectivity-app/go.mod

```diff
@@ -0,0 +1,40 @@
+module outline_sdk_connectivity_test
+
+go 1.20
+
+require (
+	github.com/Jigsaw-Code/outline-sdk v0.0.2
+	github.com/Jigsaw-Code/outline-sdk/x v0.0.0-20230807220427-893de7fdc6b8
```

Comment:
Update it to the latest main. I think you can just replace the version with `main` and call `go mod tidy`